test/test_torch.py

```diff
@@ -18433,6 +18433,93 @@ def test_atleast(self, device, dtype):
         self._test_atleast_dim(torch.atleast_2d, np.atleast_2d, device, dtype)
         self._test_atleast_dim(torch.atleast_3d, np.atleast_3d, device, dtype)
 
+    @onlyOnCPUAndCUDA
```

Comment:
I've added this test to skip on xla side, would you mind removing this decorator? (Ideally we do want backends to check against this corner case in the long term).

Reply:
Makes sense. Done! Thank You!